cogboard-webapp/src/components/WidgetForm/index.js

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from 'react';
+import { string, number, bool } from 'prop-types';
+import { useSelector } from 'react-redux';
+
+import widgetTypes from '../widgets';
+import { useFormData } from '../../hooks';
+import { sortByKey } from "../helpers";
+import { COLUMNS_MIN, ROWS_MIN } from '../../constants';
+
+import { Box, FormControlLabel, FormControl, MenuItem, TextField, Switch, Tab } from '@material-ui/core';
+import DropdownField from '../DropdownField';
+import WidgetTypeForm from '../WidgetTypeForm';
+import { StyledNumberField, StyledTabPanel, StyledTabs } from './styled';
+import { StyledFieldset } from '../styled';
+
+const renderWidgetTypesMenu = (widgetTypes) =>
```

Comment:
Personally I would move this function expression inside the component function body or to separate `helpers` file in the same directory

Reply:
Move it to component helpers.

Reply:
done :)